cmd/notifiers.go

```diff
@@ -66,12 +68,128 @@ type notifier struct {
 	// Add new notification queues.
 }
 
+// SetAMQPByID -
+func (n *notifier) SetAMQPByID(accountID string, amqpn amqpNotify) {
+	n.Lock()
+	defer n.Unlock()
+	n.AMQP[accountID] = amqpn
+}
+
+func (n *notifier) GetAMQP() map[string]amqpNotify {
+	return n.AMQP
```

Comment:
Can we hold a lock here and return a copy of AMQP ? the same of other types of notification.